example-playbooks/manage_users/README.md

```diff
@@ -0,0 +1,49 @@
+# Manage users
+
+This is an ansible playbook that allows the creation of cql users and deletes the default cassandra user.
+
+## Prerequisites
+
+* A scylla cluster with authentication enabled.
+
+## Parameters
+
+* `scylla_admin_username`: The username of the superuser which will be replacing the default cassandra user.
+* `users`: A map with the users (besides `scylla_admin_username`) which are going to be added.
+           Every user is a map containing the following entries:
+    * `superuser` -> a boolean indicating if the new user is a superuser
+    * `permissions` (optional) -> A list with the permissions. Every permission is represented by a list
+                                  with 2 positions, where the first position is the name of the permission
+                                  and the second position is the resource for which this permission is being added.
+                                  A list with the available permissions can be found in {https://docs.scylladb.com/stable/operating-scylla/security/authorization.html#permissions}.
+
+By default, this playbook will create the users `scylla_admin` (superuser), `scylla_cql_monitor` and `customer_admin`.
+Their default permissions are listed in `vars/main.yml`.
+You can override the variable `users` in order to add a different list of users with a different set of permissions.
+
+Every user defined in this playbook must be also defined in the inventory file under the section `[cql_credentials]` followed by its password.
```

Comment:
`inventory.ini` purpose is storing and grouping hosts that will be used by Ansible. Putting extra information there doesn't seems to be right at all. That's why `main.yml` exists.